mythril/analysis/modules/delegatecall.py

```diff
@@ -78,13 +78,18 @@ def _concrete_call(
         address=address,
         swc_id=DELEGATECALL_TO_UNTRUSTED_CONTRACT,
         bytecode=state.environment.code.bytecode,
-        title="Call data forwarded with delegatecall()",
-        _type="Informational",
+        title="Delegatecall Proxy",
+        severity="Low",
+        description_head="The contract implements a delegatecall proxy.",
+        description_tail="The smart contract forwards the received calldata via delegatecall. Note that callers"
+        "can execute arbitrary functions in the callee contract and that the callee contract "
+        "can access the storage of the calling contract. "
+        "Make sure that the callee contract is audited properly.",
         gas_used=(state.mstate.min_gas_used, state.mstate.max_gas_used),
     )
 
     issue.description = (
-        "This contract forwards its call data via DELEGATECALL in its fallback function. "
+        ""
```

Comment:
This line can be deleted without any side effect, right?

Reply:
Deleted.